EIPS/eip-152.md

```diff
@@ -0,0 +1,140 @@
+---
+eip: 152
+title: Add Blake2 compression function `F` precompile
+author: Tjaden Hess <tah83@cornell.edu>, Matt Luongo (@mhluongo), Piotr Dyraga (@pdyraga)
+discussions-to: https://ethereum-magicians.org/t/blake2b-f-precompile/3157
```

Comment:
Isn't https://github.com/ethereum/EIPs/issue/152 the one with the most conversation?

Reply:
Frankly it's spread across 4 significant GitHub PRs and issues, so I thought the conversation that renewed this effort was the better choice. Open to whichever and including the rest as references

Reply:
The most technical discussions were on #152. Should have just stayed there :wink:

It is confusing now, of course it up to the EIP author to decide.

Reply:
On second thought, I agree- switched it out